### PR TITLE
Add Community page

### PR DESCRIPTION
**Pull request by kylefraser**
This PR adds a new Trino Community page. 

The goal of this page is to highlight different aspects of the Trino Community such as the Slack community, Trino Community Broadcast, Meetups, Social accounts, and how users can help contribute to the project. 

Worked closely with @bitsondatadev 

<details>
<summary>Screenshot</summary>

screencapture-localhost-8888-community-2021-04-12-10_46_39

</details>

### CODE REVIEW

**Comment by martint** [comment]
This looks great! Thanks for working on it!

In the list of slack channels, I would remove `#triage`, `#releases` and `#infrastructure`. Those are low traffic channels and tend not to be very relevant to people first joining the project. It would also be great to have a link to https://trino.io/slack.html or embed the "join slack" button somewhere. Otherwise, there's no easy way for someone that wants to join those channels to know where to go to join the slack workspace first.

File: community.md [hunk 1]
```diff
@@ -0,0 +1,143 @@
...
+  <div class="col-md-12 spacer-60">
+    <div class="community-content-section">
+      <div class="community-content-container slack-content-container">
+        <div class="slack-text-container">
+          <h2>Join the discussion</h2>
+          <p>The community is very active and helpful on Slack, with users and developers from all around the world. If you need help using or running Trino, this is the place to ask.</p>
+        </div>
+        <div class="slack-stats-container">
+          <div class="community-stats">
+            <span>3884</span>
```
Comment (bitsondatadev) [inline community.md:26]: Lets update this to 3933 before we merge

File: community.md [hunk 2]
```diff
@@ -0,0 +1,143 @@
...
+            </div>
+          </div>
+          <div class="contribute-thirds-container">
+            <div class="circle-icon-container circle-pr">
+              <i class="fa fa-code-branch fa-lg"></i>
+            </div>
+            <div>
+              <h3>Open a pull request</h3>
+              <p>Improve exisiting functionality or even add your own feature.</p>
+              <a href="http://localhost:4000/development/process.html">Read about our contribution process to learn more...</a>
```
Comment (bitsondatadev) [inline community.md:135]: This shouldn't be localhost:4000.

**Review by bitsondatadev — CHANGES_REQUESTED** [review]
Few small changes

**Comment by bitsondatadev** [comment]
> This looks great! Thanks for working on it!
> 
> In the list of slack channels, I would remove `#triage`, `#releases` and `#infrastructure`. Those are low traffic channels and tend not to be very relevant to people first joining the project. It would also be great to have a link to https://trino.io/slack.html or embed the "join slack" button somewhere. Otherwise, there's no easy way for someone that wants to join those channels to know where to go to join the slack workspace first.

@martint are there any other channels we could put in their place? Maybe  `#kubernetes`,  `#iceberg`? We want to have a little bit of illusion like there's a lot going on and removing 3 leaves us with just 4 channels. One option could be to replace the current  `#general` block with the  [https://trino.io/slack.html](https://trino.io/slack.html) link, and we can make  `#general` another big block, and add the two above? I think Kubernetes and Iceberg are good for now?

File: community.md [hunk 3]
```diff
@@ -0,0 +1,143 @@
...
+      <div class="community-content-container contribute-content-container">
+        <div class="contribute-text-container">
+          <div class="contribute-github-button d-flex flex-row">
+            <a class="github-button" href="https://github.com/trinodb" data-size="large" aria-label="Follow @trinodb on GitHub">Follow @trinodb</a>
+            <a class="github-button" href="https://github.com/trinodb/trino" data-icon="octicon-star" data-size="large" data-show-count="true" aria-label="Star trinodb/trino on GitHub">Star</a>
+          </div>
+          <p>Help build the next generation of fast distributed SQL query engines for big data analytics. Trino is maintained by hundreds of around the globe.</p>
+          <p>Over the years, the Trino project has developed an extensive vision of the software and a strong development philosophy.</p>
+          <p>Understanding these helps you get up to speed on the software and how the project works quickly.</p>
+          <p><a href="">Read more about the project vision and development philosophy...</a></p>
```
Comment (bitsondatadev) [inline community.md:106]: Looks like we're missing a link here. Not sure how I missed that :)

**Review by bitsondatadev — CHANGES_REQUESTED** [review]
Last change I see needed.

File: community.md (hunk 3 above)
Comment (kylefraser) [reply community.md]: nice find!

**Review by bitsondatadev — APPROVED** [review]
Looks good now. @martint you want to take a look?

**Comment by martint** [comment]
One more comment: I would rephrase "Total Users" as "Slack Members".

File: community.md [hunk 4]
```diff
@@ -0,0 +1,143 @@
...
+  </div>
+</div>
+
+<div class="container">
+  <div class="col-md-12 spacer-60">
+    <div class="community-content-section">
+      <div class="community-content-container slack-content-container">
+        <div class="slack-text-container">
+          <h2>Join the discussion</h2>
+          <p>The community is very active and helpful on Slack, with users and developers from all around the world. If you need help using or running Trino, this is the place to ask.</p>
```
Comment (m57lyra) [inline community.md:22]: The Trino community is...

File: community.md [hunk 5]
```diff
@@ -0,0 +1,143 @@
...
+    </div>
+    <div class="community-content-section">
+      <h2>Contribute</h2>
+      <div class="community-content-container contribute-content-container">
+        <div class="contribute-text-container">
+          <div class="contribute-github-button d-flex flex-row">
+            <a class="github-button" href="https://github.com/trinodb" data-size="large" aria-label="Follow @trinodb on GitHub">Follow @trinodb</a>
+            <a class="github-button" href="https://github.com/trinodb/trino" data-icon="octicon-star" data-size="large" data-show-count="true" aria-label="Star trinodb/trino on GitHub">Star</a>
+          </div>
+          <p>Help build the next generation of fast distributed SQL query engines for big data analytics. Trino is maintained by hundreds of around the globe.</p>
```
Comment (m57lyra) [inline community.md:103]: - fast, distributed
- hundreds of people

File: community.md [hunk 6]
```diff
@@ -0,0 +1,143 @@
...
+            </div>
+          </div>
+          <div class="contribute-thirds-container">
+            <div class="circle-icon-container circle-pr">
+              <i class="fa fa-code-branch fa-lg"></i>
+            </div>
+            <div>
+              <h3>Open a pull request</h3>
+              <p>Improve exisiting functionality or even add your own feature.</p>
+              <a href="../development/process.html">Read about our contribution process to learn more...</a>
```
Comment (m57lyra) [inline community.md:135]: Read about our <a href="../development/process.html">contribution process</a>  to learn more.

**Review by m57lyra — APPROVED** [review]
Just a few nitpicks.

File: community.md [hunk 7]
```diff
@@ -0,0 +1,143 @@
...
+layout: default
+menu_id: community
+title: Trino Community
+---
+
+<div class="jumbotron card card-image" style="background-image: linear-gradient(to top, #001C93 0%, #000033 70%);background-size:cover;">
+  <div class="text-white text-center py-5 px-4">
+    <div>
+      <h1><a href="/community/">Community</a></h1>
+      <p class="lead">Join thousands of users and developers from all over the world. </p>
```
Comment (mosabua) [inline community.md:11]: from all around the world

File: community.md [hunk 8]
```diff
@@ -0,0 +1,143 @@
...
+    <div class="community-content-section">
+      <div class="community-content-container slack-content-container">
+        <div class="slack-text-container">
+          <h2>Join the discussion</h2>
+          <p>The Trino community is very active and helpful on Slack, with users and developers from all around the world. If you need help using or running Trino, this is the place to ask.</p>
+        </div>
+        <div class="slack-stats-container">
+          <div class="community-stats">
+            <span>3933</span>
+            Slack Members
```
Comment (mosabua) [inline community.md:27]: Slack members

File: community.md [hunk 9]
```diff
@@ -0,0 +1,143 @@
...
+          </div>
+          <div class="community-stats">
+            <span>507</span>
+            Weekly Active
```
Comment (mosabua) [inline community.md:31]: Weekly active members

File: community.md [hunk 10]
```diff
@@ -0,0 +1,143 @@
...
+          <div class="community-stats">
+            <span>454</span>
+            Contributors
+          </div>
+        </div>
+      </div>
+      <div class="slack-channels-top d-flex mb-3">
+        <a href="https://join.slack.com/t/trinodb/shared_invite/zt-h88upx5f-gVb8XUu16W_jo4I2Cw8ijA" class="slack-channel slack-blue-gradient">
+          <img src="../assets/images/graphics/community-general.svg">
+          Join Slack
```
Comment (mosabua) [inline community.md:42]: Join Trino Slack

or 

Join Trino community

File: community.md [hunk 11]
```diff
@@ -0,0 +1,143 @@
...
+        </a>
+      </div>
+      <div class="slack-channels-bottom d-flex">
+        <a href="https://trinodb.slack.com/archives/CP1MUNEUX" class="slack-channel slack-orange-gradient">
+          <i class="fa fa-code fa-lg"></i>
+          #dev
+        </a>
+        <a href="https://trinodb.slack.com/archives/CFSDUNJ4W" class="slack-channel slack-pink-gradient">
+          <i class="fa fa-dharmachakra fa-lg"></i>
+          #kubernetes
```
Comment (mosabua) [inline community.md:60]: I would leave this channel out since we dont have any useful k8s offering from the project.

File: community.md [hunk 12]
```diff
@@ -0,0 +1,143 @@
...
+        </a>
+        <a href="https://trinodb.slack.com/archives/CJ6UC075E" class="slack-channel slack-blue-gradient">
+          <i class="fa fa-server fa-lg"></i>
+          #iceberg
```
Comment (mosabua) [inline community.md:64]: I would leave this one out as well.. too specific

File: community.md [hunk 13]
```diff
@@ -0,0 +1,143 @@
...
+        </a>
+        <a href="https://trinodb.slack.com/archives/CFQAMGRQE" class="slack-channel slack-pinkorange-gradient">
+          <img src="../assets/images/graphics/community-announcements.svg">
+          <i class="fa fa-bullhorn fa-lg"></i>
+          #community-announcements
```
Comment (mosabua) [inline community.md:69]: Maybe add releases channel

File: community.md [hunk 14]
```diff
@@ -0,0 +1,143 @@
...
+        </a>
+      </div>
+    </div>
+    <div class="community-content-section">
+      <h2>Events that don’t suck</h2>
+      <div class="community-content-container">
+        <a href="./broadcast" class="community-broadcast-block">
+          <div class="community-broadcast-image-wrapper">
+            <img src="../assets/images/community/tcb.png" />
```
Comment (mosabua) [inline community.md:78]: alt text for image

File: community.md [hunk 15]
```diff
@@ -0,0 +1,143 @@
...
+            <img src="../assets/images/community/tcb.gif" />
+          </div>
+        </a>
+        <div class="events-thirds-container">
+          <a href="https://www.meetup.com/topics/trino/" class="event-third-block"></a>
+          <a href="https://twitter.com/trinodb" class="event-third-block">
+            <div>Follow us<h2>@trinodb</h2></div>
+            <img src="../assets/images/community/twitter.png" />
```
Comment (mosabua) [inline community.md:86]: alt text for image

File: community.md [hunk 16]
```diff
@@ -0,0 +1,143 @@
...
+          </div>
+          <div class="community-stats">
+            <span>454</span>
+            Contributors
+          </div>
+        </div>
+      </div>
+      <div class="slack-channels-top d-flex mb-3">
+        <a href="https://join.slack.com/t/trinodb/shared_invite/zt-h88upx5f-gVb8XUu16W_jo4I2Cw8ijA" class="slack-channel slack-blue-gradient">
+          <img src="../assets/images/graphics/community-general.svg">
```
Comment (mosabua) [inline community.md:41]: alt text for image

File: community.md [hunk 17]
```diff
@@ -0,0 +1,143 @@
...
+        </a>
+        <div class="events-thirds-container">
+          <a href="https://www.meetup.com/topics/trino/" class="event-third-block"></a>
+          <a href="https://twitter.com/trinodb" class="event-third-block">
+            <div>Follow us<h2>@trinodb</h2></div>
+            <img src="../assets/images/community/twitter.png" />
+          </a>
+          <a href="https://www.youtube.com/c/trinodb" class="event-third-block">
+            <div>Catch up on what's happening in the Trino community</div>
+            <img src="../assets/images/community/youtube.png" />
```
Comment (mosabua) [inline community.md:90]: alt text for image

File: community.md [hunk 18]
```diff
@@ -0,0 +1,143 @@
...
+          </div>
+        </a>
+        <div class="events-thirds-container">
+          <a href="https://www.meetup.com/topics/trino/" class="event-third-block"></a>
+          <a href="https://twitter.com/trinodb" class="event-third-block">
+            <div>Follow us<h2>@trinodb</h2></div>
+            <img src="../assets/images/community/twitter.png" />
+          </a>
+          <a href="https://www.youtube.com/c/trinodb" class="event-third-block">
+            <div>Catch up on what's happening in the Trino community</div>
```
Comment (mosabua) [inline community.md:89]: Make this more about videos somehow..

File: community.md [hunk 19]
```diff
@@ -0,0 +1,143 @@
...
+          </a>
+        </div>
+      </div>
+    </div>
+    <div class="community-content-section">
+      <h2>Contribute</h2>
+      <div class="community-content-container contribute-content-container">
+        <div class="contribute-text-container">
+          <div class="contribute-github-button d-flex flex-row">
+            <a class="github-button" href="https://github.com/trinodb" data-size="large" aria-label="Follow @trinodb on GitHub">Follow @trinodb</a>
```
Comment (mosabua) [inline community.md:100]: maybe need to say on github .. so its not confusing with following on twitter?

File: community.md [hunk 20]
```diff
@@ -0,0 +1,143 @@
...
+            <a class="github-button" href="https://github.com/trinodb/trino" data-icon="octicon-star" data-size="large" data-show-count="true" aria-label="Star trinodb/trino on GitHub">Star</a>
```
Comment (mosabua) [inline community.md:101]: Maybe text needs to mention Star Trino on GitHub?

File: community.md [hunk 21]
```diff
@@ -0,0 +1,143 @@
...
+          </div>
+          <p>Help build the next generation of fast, distributed SQL query engines for big data analytics. Trino is maintained by hundreds of people around the globe.</p>
```
Comment (mosabua) [inline community.md:103]: why are these three sentences separate paragraphs?

File: community.md [hunk 22]
```diff
@@ -0,0 +1,143 @@
...
+          <p><a href="../development/vision.html">Read more about the project vision and development philosophy.</a></p>
+        </div>
+        <div class="d-flex flex-column" style="flex: 1 1 50%;">
+          <div class="contribute-thirds-container">
+            <div class="circle-icon-container circle-bugs">
+              <i class="fa fa-bug fa-lg"></i>
+            </div>
+            <div>
+              <h3>Find and report issues</h3>
+              <p>Report bugs on Github and report security vulnerabilities to <a href="mailto:security@trino.io">security@trino.io</a></p>
```
Comment (mosabua) [inline community.md:115]: Break up into separate bug and security issue items

Comment (mosabua) [reply community.md:115]: also generally.. its GitHub

File: community.md [hunk 23]
```diff
@@ -0,0 +1,143 @@
...
+              <a class="github-button" href="https://github.com/trinodb/trino/issues" data-icon="octicon-issue-opened" data-size="large" data-show-count="true" aria-label="Issue trinodb/trino on GitHub">Issue</a>
+            </div>
+          </div>
+          <div class="contribute-thirds-container">
+            <div class="circle-icon-container circle-docs">
+              <i class="fa fa-pencil-alt fa-lg"></i>
+            </div>
+            <div>
+              <h3>Write documentation</h3>
+              <p>Contributions are not just about code. Help the community understand how to use our distributed SQL engines by adding to <a href="../docs/current/">our docs</a>.</p>
```
Comment (mosabua) [inline community.md:125]: Maybe add a link to the readme? https://github.com/trinodb/trino/tree/master/docs

File: community.md [hunk 24]
```diff
@@ -0,0 +1,143 @@
...
+            </div>
+          </div>
+          <div class="contribute-thirds-container">
+            <div class="circle-icon-container circle-pr">
+              <i class="fa fa-code-branch fa-lg"></i>
+            </div>
+            <div>
+              <h3>Open a pull request</h3>
+              <p>Improve exisiting functionality or even add your own feature.</p>
+              <a href="../development/process.html">Read about our contribution process to learn more.</a>
```
Comment (mosabua) [inline community.md:135]: Add link to developer guide in addition here? https://trino.io/docs/current/develop.html

**Review by mosabua — CHANGES_REQUESTED** [review]
A bunch of tweaks necessary

File: community.md (hunk 19 above)
Comment (kylefraser) [reply community.md:96]: These render as GitHub buttons with the GitHub icon.

File: community.md (hunk 20 above)
Comment (kylefraser) [reply community.md:97]: This renders as a GitHub button with a Star icon.

**Review by mosabua — APPROVED** [review]
Looks good now. Let's ship it!

**Pull request merged**